src/store/modules/ADempiere/pointOfSales/payments/mutations.js

```diff
@@ -57,5 +57,12 @@ export default {
   },
   setFieldCurrency(state, currency) {
     state.fieldCurrency = currency
+  },
+  dialogoInvoce(state, { show, type, success }) {
```

Comment:
please change mutation name to `dialogInvoice` (without the `o` at the end of the dialog)